gui/src/app/state/recovery.rs

```diff
@@ -247,7 +247,7 @@ fn recovery_paths(wallet: &Wallet, coins: &[Coin], blockheight: i32) -> Vec<Reco
                 .iter()
                 .filter(|coin| {
                     coin.spend_info.is_none()
-                        && remaining_sequence(coin, blockheight as u32, sequence) <= 1
+                        && remaining_sequence(coin, blockheight as u32, sequence, true) <= 1
```

Comment:
I think it might be enough to include an additional filter here that `coin.block_height.is_some()` and then the number of recoverable coins will match those found by the `createrecovery` command. Would this avoid the need to modify the `remaining_sequence` function?